### PR TITLE
Monitor also component labels in configmaps

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: thoth/metrics_exporter/jobs.py [hunk 1]
```diff
@@ -119,21 +119,21 @@ def count_configmaps(config_map_list_items: list) -> int:
...
-        for operator in operators:
-            config_maps_items = openshift.get_configmaps(namespace=namespace, label_selector=operator)
+        for label in labels:
+            config_maps_items = openshift.get_configmaps(namespace=namespace, label_selector=label)
             number_configmaps = count_configmaps(config_maps_items)
-            metrics.config_maps_number.labels(namespace, operator).set(number_configmaps)
+            metrics.config_maps_number.labels(namespace, labels).set(number_configmaps)
             _LOGGER.debug(
-                "thoth_config_maps_number=%r, in namespace=%r for label=%r", number_configmaps, namespace, operator
+                "thoth_config_maps_number=%r, in namespace=%r for label=%r", number_configmaps, namespace, labels
```
Comment (pacospace) [inline thoth/metrics_exporter/jobs.py]: typo ```label``` not ```labels```  under the second for cycle otherwise LGTM! Thanks @fridex

Comment (fridex) [reply thoth/metrics_exporter/jobs.py]: Nice catch! Fixed. Thanks! :+

File: thoth/metrics_exporter/jobs.py [hunk 2]
```diff
@@ -119,21 +119,21 @@ def count_configmaps(config_map_list_items: list) -> int:
...
     for namespace in namespaces:
         _LOGGER.info("Evaluating configmaps metrics for Thoth namespace: %r", namespace)
 
-        for operator in operators:
-            config_maps_items = openshift.get_configmaps(namespace=namespace, label_selector=operator)
+        for label in labels:
+            config_maps_items = openshift.get_configmaps(namespace=namespace, label_selector=label)
             number_configmaps = count_configmaps(config_maps_items)
-            metrics.config_maps_number.labels(namespace, operator).set(number_configmaps)
+            metrics.config_maps_number.labels(namespace, labels).set(number_configmaps)
```
Comment (pacospace) [inline thoth/metrics_exporter/jobs.py]: Also here typo for ```labels```, it should be ```label```

Comment (fridex) [reply thoth/metrics_exporter/jobs.py]: :angel:

**Pull request merged**